src/Grpc/Server.php

```
@@ -136,17 +167,69 @@ public function serve(?WorkerInterface $worker = null, ?callable $finalize = nul
+        $interceptors = $this->interceptors[$serviceName] ?? [];
+
+        if (empty($interceptors)) {
+            return $service->invoke($method, $context, $body);
+        }
+
+        $interceptorInstances = [];
+        foreach ($interceptors as $interceptor) {
+            $interceptorInstances[] = $this->createInterceptor($interceptor);
+        }
+
+        $handler = function (CallContextInterface $_ctx) use ($service, $method, $context, $body) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$handler'. (undefined)

(UnusedLocalVariable)